Simple-ML/Simple-ML
# fix: scrollbars

Closes #[issue_no].

## Summary of Changes

Fixed scroll behavior, added new context menu icons, added filter function to context menu 

## Screenshots (if necessary)


![Screenshot 2022-04-22 at 19 04 02](https://user-images.githubusercontent.com/102034120/164761327-4771d4cc-bbbd-4061-8d69-1b3bc8c66cbf.png)

## Testing instructions

check scrollbars in the graphical editor, check new icons in the context menu, and test search function in the context menu
